lib/src/components/card_rounded.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+
+/// This is a card widget with a rounded border, that has optional close button
+/// at the top right.
+class CardRounded extends StatelessWidget {
+  /// A child widget for [Card].
+  final Widget? child;
+
+  /// A variable to set the background color of the card.
```

Comment:
Let's be more consice, here and in all other places.
```suggestion
  /// The background color of the card.
```